### PR TITLE
DCS-1611 Patching, bumping circle orb, java version and standardising…

### PR DESCRIPTION
**Pull request by andrewrlee**
… health components

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ workflows:
               ignore: /.*/
       - hmpps/helm_lint:
           name: helm_lint
-      - hmpps/build_docker:
+      - hmpps/build_multiplatform_docker:
```
Comment (andrewrlee) [inline .circleci/config.yml:46]: moving to multi platform build which should work for new M1 macs

File: src/main/java/uk/gov/justice/hmpps/probationteams/health/HealthInfo.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+/**
+ * Adds version data to the /health endpoint. This is called by the UI to display API details
+ */
+@Component
+class HealthInfo(buildProperties: BuildProperties) : HealthIndicator {
+  private val version: String = buildProperties.version
+
+  override fun health(): Health = Health.up().withDetail("version", version).build()
+}
```
Comment (andrewrlee) [inline src/main/java/uk/gov/justice/hmpps/probationteams/health/HealthInfo.kt:16]: We've got a bug where we aren't logging version number on start up.

Moving to make this more consistent with the version in the kotlin template in the hope the bug might fall out

File: src/test/java/uk/gov/justice/hmpps/probationteams/health/HealthInfoTest.kt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+class HealthInfoTest {
+  @Test
+  fun `should include version info`() {
+    val properties = Properties()
+    properties.setProperty("version", "somever")
+    assertThat(HealthInfo(BuildProperties(properties)).health().details)
+      .isEqualTo(mapOf("version" to "somever"))
+  }
+}
```
Comment (andrewrlee) [inline src/test/java/uk/gov/justice/hmpps/probationteams/health/HealthInfoTest.kt:16]: copied over from template project

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**